pkg/middlewares/aws_security_group_rules_defaults.go

```diff
@@ -0,0 +1,70 @@
+package middlewares
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+// When scanning a brand new AWS account, some users may see irrelevant results about default AWS security group rules.
+// We ignore these resources by default when strict mode is disabled.
+type AwsSecurityGroupRuleDefaults struct{}
```

Comment:
I think this is already handled too

Reply:
It's handled implicitly since we do ignore the default security group.

Reply:
So should I delete this middleware? It's unclear to me what you mean by "this is already handled" since users see default security group rules as drifted resources in their scan result.

Reply:
Can you reproduce this behaviour where default sg rules are seem as drifted ? I'm not sure of that use case

Reply:
@wbeuil it has been seen in that video, referenced in the issue  https://github.com/cloudskiff/driftctl/issues/331#issuecomment-808262088